src/Pages/Templates/TemplateDetails/TemplateDetails.tsx

```diff
@@ -0,0 +1,138 @@
+  titleWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    padding: '24px 0',
+    justifyContent: 'space-between',
+  },
+  labelGroup: {
+    marginLeft: '8px',
+  },
+  descriptionMaxWidth: {
+    maxWidth: '1600px',
+  },
+  childContainer: {
+    margin: '24px',
+    background: global_BackgroundColor_light_100.value,
+  },
+});
+
+export default function TemplateDetails() {
+  const classes = useStyles();
+  const { templateUUID } = useParams();
+
+  const rootPath = useRootPath();
+  const navigate = useNavigate();
+
+  const { data, isError, error, isLoading } = useFetchTemplate(templateUUID as string);
+
+  const {
+    isError: repositoryParamsIsError,
+    isLoading: archVersionLoading,
+    error: repositoryParamsError,
+    archesDisplay,
+    versionDisplay,
+  } = useArchVersion();
+
+  // Error is caught in the wrapper component
+  if (isError) throw error;
+  if (repositoryParamsIsError) throw repositoryParamsError;
+
+  const navigateToTemplateList = () => navigate(rootPath + `/${TEMPLATES_ROUTE}`);
+
+  if (isLoading || archVersionLoading) {
+    return <Loader />;
+  }
+
+  return (
+    <>
+      <Grid className={classes.topContainer}>
+        <Stack>
+          <StackItem>
+            <Breadcrumb ouiaId='template_details_breadcrumb'>
+              <BreadcrumbItem component='button' onClick={navigateToTemplateList}>
+                Templates
+              </BreadcrumbItem>
+              <BreadcrumbItem disabled>{data?.name}</BreadcrumbItem>
+            </Breadcrumb>
+          </StackItem>
+          <StackItem className={classes.titleWrapper}>
+            <Flex
+              direction={{ default: 'row' }}
+              justifyContent={{ default: 'justifyContentCenter' }}
+            >
+              <Title headingLevel='h1'>{data?.name}</Title>
+              <LabelGroup className={classes.labelGroup}>
+                <Label isCompact color='blue'>
+                  {versionDisplay(data?.version)}
+                </Label>
+                <Label isCompact color='blue'>
+                  {archesDisplay(data?.arch)}
+                </Label>
+              </LabelGroup>
+            </Flex>
+            <TemplateActionDropdown />
+          </StackItem>
+          <StackItem className={classes.descriptionMaxWidth}>
+            <Flex
+              direction={{ default: 'row' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <Flex direction={{ default: 'column' }}>
+                <DetailItem title='Description:' value={data?.description} />
+                <DetailItem
+                  title='Includes content up to:'
+                  value={data?.date ? formatDateDDMMMYYYY(data.date) : ''}
+                />
+              </Flex>
+              <Flex direction={{ default: 'column' }}>
+                <DetailItem title='Created by:' value={data?.created_by} />
+                <DetailItem title='Created:' value={data?.created_at} />
```

Comment:
i think there should be some conditional date formatting here similar to what you have for `date` and `updated_at`

Reply:
Good shout, checking.